### DISCUSSION TITLE
Project Name

### DISCUSSION THREAD

**Discussion by OnceUponALoop** [Open-Enterprise-OS/oeos#6] [General]
Open Enterprise OS was just the first thing that came to my mind.

We'd need to vote on a replacement name as a community.

A few people suggested the following names in the slack channel

- Open Enterprise OS (OEOS)
- Second Community Enterprise Operating System (ScentOS)
- Float Enterprise Linux (FEL)

I agree with this comment from Stephen LePage
> From a naming perspective, I would like to see emphasis on the Community aspect.

**Comment by OnceUponALoop** [comment]
Previous comments in the now closed issue #1

**Reply by hbjydev** [reply]
I think ScentOS is a little too close to CentOS personally. Might bodge any SEO we get.

I also think we should keep EL in the name, to ensure people could easily throw it in the same ballpark as RHEL, perhaps even assume it's a RHEL derivative right from the name.

---

OpenEL was suggested on slack.

---

Open Enterprise Linux would mismatch Oracle Enterprise Linux (another rhel clone)

---

> Open Enterprise Linux would mismatch Oracle Enterprise Linux (another rhel clone)

I understand how that can be conflated, but how though? CentOS isn't named COS. OpenEL would be the name, and Open and Oracle do not match.

Edit: Doesn't matter, this is already taken: https://github.com/openel

---

It's a shame OpenEL is taken, that would be a pretty good distro name for this imo

---

Kurt in Slack suggested "Delta", because that's where the stream ends.

**Reply by ghost** [reply]
what about Core Ent Linux > CEL ?

**Comment by hbjydev** [comment]
(Bringing this back from the old issue)

Kurt in Slack suggested "Delta", because that's where the stream ends.

**Reply by OnceUponALoop** [reply]
> Kurt in Slack suggested "Delta", because that's where the stream ends.
Becomes an issue with Dell from an SEO and general confusion standpoint, just my 2c

**Reply by hbjydev** [reply]
@OnceUponALoop Can you elaborate on that? As pointed out in Slack, I think there'll always be collisions with SEO. But "Delta" and "DELL" are less likely to cause collisions than "OEL" and "OL", for example.

**Comment by hbjydev** [comment]
I hold the strong opinion that "EL" should be somewhere in the name, indicating it's still an Enterprise Linux distro, and perhaps even lead people to come to the "It's a RHEL derivative" conclusion straight from the name.

**Comment by hemzaz** [comment]
what about HELIOS - HPC & Enterprise LInux  OS

**Reply by oferchen** [reply]
+1

**Reply by hydrosIII** [reply]
What does it stands for?

**Reply by hemzaz** [reply]
@hydrosIII 
HELIOS = SUN and initials for - HPC & Enterise LInux OS

**Comment by GlacierFox** [comment]
Cintra

**Comment by bluikko** [comment]
NouvEL, pronounced like "novel". 'cause french is sophisticated.
